www/docs/guides/use_ERC20.md

```diff
@@ -56,21 +56,21 @@ Declaration and deployment of the ERC20 contract:
 ```typescript
 // Deploy an ERC20 contract
 console.log("Deployment Tx - ERC20 Contract to Starknet...");
-const compiledErc20mintable = json.parse(fs.readFileSync("compiled_contracts/ERC20MintableOZ051.json").toString("ascii"));
-    const initialTk: Uint256 = cairo.uint256(100);
-    const erc20CallData: CallData = new CallData(compiledErc20mintable.abi);
+const compiledSierra = json.parse(fs.readFileSync("./compiled_contracts/erc20OZ081.sierra.json").toString("ascii"));
```

Comment:
wdyt about removing this ./compiled_contracts and instead take everything we need from __mocks__ ?
cc: @penovicp @tabaktoni